### ISSUE TITLE
about build error

### ISSUE THREAD

**Issue by wengqi123** [ndaniel/fusioncatcher#73]
use command：
```
./bin/fusioncatcher-build -g homo_sapiens -o /haplox/users/wenger/pipe/RNA/fusion_human 

error ：
Downloading the viral genomes from NCBI server!
230 Anonymous access granted, restrictions apply
ERROR: File not 'all.fna.tar.gz' found!
```

**Comment by ndaniel** [comment]
The error is due to NCBI changing the URL for the viruses genomes and also the files names on October 20, 2017.

The newer version of  https://github.com/ndaniel/fusioncatcher/blob/master/bin/get_viruses.py should fix this.

This will be fixed permanently in the next release of FusionCatcher that is v0.99.7e

**Comment by wengqi123** [comment]
ok ,tahnks!

**Comment by ndaniel** [comment]
fixed in v1.00
